components/card-list/css/_mixin.scss

```diff
@@ -0,0 +1,52 @@
+@mixin utrecht-card-list {
+  display: flex;
+  flex-wrap: wrap;
+  margin-block-end: calc(
+    var(--utrecht-space-around, 0) *
+      var(--utrecht-card-list-margin-block-end, var(--utrecht-paragraph-margin-block-end, 0))
+  );
+  margin-block-start: calc(
+    var(--utrecht-space-around, 0) *
+      var(--utrecht-card-list-margin-block-start, var(--utrecht-paragraph-margin-block-start, 0))
+  );
+  padding-inline-start: var(--utrecht-card-list-padding-inline-start, 0);
+}
+
+@mixin utrecht-card-list__item {
+  block-size: 100%;
+  inline-size: 100%;
+  list-style: none;
+  margin-block-end: var(--utrecht-card-list-item-margin-block-end, 1em);
+  margin-block-start: var(--utrecht-card-list-item-margin-block-start);
+  max-inline-size: var(--utrecht-card-list-item-max-inline-size, 350px);
+  min-block-size: 312px;
+  padding-inline-start: var(--utrecht-card-list-item-padding-inline-start, 1em);
```

Comment:
Kun je de hardcoded waarden vervangen door enkele CSS-variabelen uit dit bestand? https://github.com/nl-design-system/utrecht/blob/main/proprietary/design-tokens/src/common/utrecht/space.tokens.json

Daarnaast moeten we deze waarden valideren met de designers van Gemeente Utrecht.